_posts/2024-08-01-contributing-to-che-docs.adoc

```diff
@@ -0,0 +1,106 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
+categories: []
+keywords: ['che-docs', 'documentation', 'write', 'contribute']
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+= Contributing to Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your own Eclipse Che installation]
```

Comment:
Let's mention it as a second option, or not mention it at all

Reply:
got it